src/js/components/PodDetail.js

```diff
@@ -90,16 +90,13 @@ class PodDetail extends mixin(TabsMixin) {
     const {activeActionDialog} = this.state;
 
     return (
-      <div className="flex-container-col">
```

Comment:
The page flex properties need to be addressed in a separate PR; the `flex-container-col` class definition doesn't exist anymore.